src/test/java/org/sagebionetworks/bridge/reporter/worker/RetentionReportGeneratorTest.java

```diff
@@ -38,8 +40,11 @@
     
     private static final DateTime STUDY_START_DATE_1 = DateTime.parse("2017-06-05T00:00:00.000Z");
     private static final DateTime STUDY_START_DATE_2 = DateTime.parse("2017-06-04T00:00:00.000Z");
+    
     private static final DateTime SIGN_IN_ON = DateTime.parse("2017-06-09T00:00:00.000Z");
-    private static final DateTime UPLOADED_ON = DateTime.parse("2017-06-09T00:00:00.000Z");
+    private static final DateTime SIGN_IN_ON_TIMEZONE = DateTime.parse("2017-06-04T06:50:21.650-07:00");
```

Comment:
We should pick a time late at night, like 18:00-0700, to make sure that converting the timezone makes a difference.